README.md

```diff
@@ -10,8 +10,8 @@ npm install remix-utils
 
 Additional optional dependencies may be needed, all optional dependencies are:
 
-- `@remix-run/react` (also `@remix-run/router` but you should be using the React one)
-- `@remix-run/node` or `@remix-run/cloudflare` or `@remix-run/deno` (actually it's `@remix-run/server-runtime` but you should use one of the others)
+- `react-router`
+- `@react-router/node` or `@react-router/cloudflare`
```

Comment:
I just saw the rest of the code, this is not needed, nothing depends on these packages